scrapers/ma/bills.py

```diff
@@ -216,6 +182,13 @@ def scrape_bill(self, session, bill_id, chamber):
         if bill_summary:
             bill.add_abstract(bill_summary, "summary")
 
+        if bill_meta["BillNumber"] and bill_meta["DocketNumber"]:
```

Comment:
In other places in the code, we rely on `bill_meta["DocketNumber"]` having a value. Why do we check for it here?

Reply:
Mirroring the states's json, these keys are always defined, and are either `String` or `None`. 

If they're both not `None`, then we want to add the relation because the bill has both numbers.

I like the way this simplifies the logic of "Bill Number if its not None, otherwise fall back docket number" into one `or` statement, but if you feel like that's leaning a bit too much on type trickery i'm happy to restructure. The alternative is probably to only define the key when it's set, and then reference it by `.get("BillNumber", ` with a default of the docket number like your previous suggestion.

Reply:
Really I'm just calling out differing behavior within the scraper. If this is the right solution, we should follow it throughout the scraper. If falling back to `DocketNumber` is the correct solution, we should do that.